<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationDialog</class>
 <widget class="QDialog" name="RegistrationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Регистрация</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="1">
          <widget class="QLineEdit" name="lne_address"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lbl_address">
           <property name="text">
            <string>Адрес</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lne_name"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_name">
           <property name="text">
            <string>Название</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="gb_coords">
         <property name="title">
          <string>Координаты</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1">
          <item row="0" column="0">
           <widget class="QLabel" name="lbl_latitude">
            <property name="text">
             <string>Широта</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="spb_latitude">
            <property name="minimum">
             <double>-90.000000000000000</double>
            </property>
            <property name="maximum">
             <double>90.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lbl_longitude">
            <property name="text">
             <string>Долгота</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="spb_longitude">
            <property name="maximum">
             <double>360.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gb_type">
         <property name="title">
          <string>Кто вы?</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="rb_supplier">
            <property name="text">
             <string>Поставщик</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_transfer">
            <property name="text">
             <string>Перевозчик</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_customer">
            <property name="text">
             <string>Заказчик</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gb_counter">
         <property name="title">
          <string>Количество пунктов</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,1">
          <item row="0" column="0">
           <widget class="QLabel" name="lbl_counter">
            <property name="text">
             <string>Склады</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>5</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="wdg_map" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RegistrationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RegistrationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
